src/preamble.js

```diff
@@ -1216,6 +1216,7 @@ function createWasm() {
         // Don't use streaming for file:// delivered objects in a webview, fetch them synchronously.
         !isFileURI(wasmBinaryFile) &&
 #endif
+        !ENVIRONMENT_IS_NODE &&
```

Comment:
```suggestion
 #if ENVIRONMENT_MAY_BE_NODE
        !ENVIRONMENT_IS_NODE &&
#endif
```
(this avoids extra code in non-node builds)